main.py

```diff
@@ -0,0 +1,12 @@
+import settings
+import os
+import subprocess
+import utils
+from time import sleep
+
+while True:
+    utils.init_datadog()
+    utils.Report.ok()
+    subprocess.Popen(
+        ["scrapy", "runspider", "cluster_spider.py"], env=os.environ)
```

Comment:
The passing of the entire environment is mainly just an artifact of the `subprocess` method of executing the scraper.

However the shouty stacktrace errors have been addressed by other changes. (an env helper to gracefully bow out on missing required vars and making setting tags optional via `DATADOG_TAGS` envvar

Reply:
Heh, ok think I misunderstood ... willfix 👍

Reply:
I was thinking about whether we need to pass over all of `os.environ` because I was finding it difficult to understand, at a high level without reading further down the code, what environment variables were required to run. It felt like they could be function arguments instead. It sounds like the env helper will help though.